recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: libgfortran-abi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+requirements:
+  run:
+    - libgfortran 3.0.0
```

Comment:
FWIW I was not successful at getting the same info from `strings` on Mac. Though I am probably missing something.

Also, to add something else to this discussion I installed `libgfortran` via `yum`. The library is versioned 3.0.0, but the ABI version is 1.2. There's certainly something I'm missing here. 😕